docs/source/models.rst

```diff
@@ -97,6 +98,9 @@ You can construct a model with random weights by calling its constructor:
     convnext_small = models.convnext_small()
     convnext_base = models.convnext_base()
     convnext_large = models.convnext_large()
+    swin_t = models.swin_t()
+    swin_s = models.swin_s()
+    swin_b = models.swin_b()
```

Comment:
This is the legacy way we document models and we should definitely add info here. In addition to this, we need to implement the new way of documenting models described at #5833

Reply:
yes I was woking on this in a separate commit - should be added now.